lib/jsdom/living/nodes/Node.idl

```diff
@@ -51,4 +51,10 @@ interface Node : EventTarget {
   Node appendChild(Node node);
   Node replaceChild(Node node, Node child);
   Node removeChild(Node child);
+
+  Node? append(sequence<any> nodes);
+  Node? prepend(sequence<any> nodes);
+  Node? after(sequence<any> nodes);
+  Node? before(sequence<any> nodes);
+  Node? replaceWith(sequence<any> nodes);
```

Comment:
thanks they was defined there, I wonder why I didn't find them

`(Node or DOMString)` when testing in chrome, it allows anything `document.body.firstChild.after(2)`, `document.body.firstChild.after({})` and just stringify